modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -21,13 +21,40 @@ object SeqexecEngine {
 
   sealed trait SeqexecEvent
   object SeqexecEvent {
-    case class SequenceStart(view: SequenceView) extends SeqexecEvent
-    case class StepExecuted(view: SequenceView) extends SeqexecEvent
-    case class SequenceCompleted(view: SequenceView) extends SeqexecEvent
-    case class StepBreakpointChanged(view: SequenceView) extends SeqexecEvent
-    case class StepSkipMarkChanged(view: SequenceView) extends SeqexecEvent
-    case class SequencePauseRequested(view: SequenceView) extends SeqexecEvent
-    case class NewLogMessage(msg: LogMsg) extends SeqexecEvent
+
+    import edu.gemini.seqexec.engine
```

Comment:
I guess I often put all the imports on the top unless there is some naming conflicts. it's a minor thing